server/routes/cookies/CookiesPolicyController.ts

```diff
@@ -0,0 +1,53 @@
+import { NextFunction, Request, Response } from 'express'
+
+export default class CookiesPolicyController {
+  async initialiseCookieSession(req: Request, res: Response, next: NextFunction): Promise<void> {
```

Comment:
What does this method do and where is/will it be called from?
I can't quite grok why there is different behaviour for external vs. internal users ?

Reply:
It's called in an app.use when setting up the cookie routes. It's basically there to save the existing cookie in the session and fill locals that can be used in views.

The difference between internal and external users is that internal users won't get the cookie banner or get a chance to opt out, as is standard across internal DPS apps.